### ISSUE TITLE
Rocket chip and freedom

### ISSUE THREAD

**Issue by NarimanMuhammed** [sifive/freedom#121]
can I use freedom E SDK to compile and debug a program on rocket-chip?

**Comment by erikdanie** [comment]
Yeah you can, check the documentation [here](https://sifive.github.io/freedom-e-sdk-docs/index.html). Any discussion is probably more suited for the [sifive forums](https://forums.sifive.com/) though, since more of the active community looks there.
